### ISSUE TITLE
libasound.so.2: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by ksaj** [sqshq/sampler#52]
$ sampler
sampler: error while loading shared libraries: libasound.so.2: cannot open shared object file: No such file or directory

$ ldconfig -p | grep asound
	libasound.so.2 (libc6) => /usr/lib/i386-linux-gnu/libasound.so.2
	libasound.so (libc6) => /usr/lib/i386-linux-gnu/libasound.so

**Comment by sqshq** [comment]
Duplicate: https://github.com/sqshq/sampler/issues/5

**Comment by sqshq** [comment]
Hi @ksaj,
Are you saying the library is available in the system, but you still have the issue?

**Comment by ksaj** [comment]
Yes. That's why I showed the ldconfig output. To clarify, the same results occur if I run it with sudo.
